OpenTaiko/src/Stages/07.Game/Taiko/CActImplJudgeText.cs

```diff
@@ -35,6 +35,7 @@ public override int Draw() {
 					var judgeC = JudgeAnimes[j][i];
 					if (judgeC.counter.CurrentValue == judgeC.counter.EndValue) {
 						JudgeAnimes[j].Remove(judgeC);
+						i--;
```

Comment:
How about:
```csharp
JudgeAnimes[j].RemoveAt(i--);
```
`List.Remove()` is $O(n)$ search ***and*** $O(n)$ data movement, while `List.RemoveAt()` is just $O(n)$ data movement.

Reply:
much better